enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/components/util/BadgeList/AddBadgeListItem/AddBadgeListItem.unit.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import userEvent from "@testing-library/user-event";
+
+import { render, screen } from "__support__/ui";
+
+import { AddBadgeListItem } from "./AddBadgeListItem";
+
+const setup = () => {
+  const name = "test badge";
+  const handleClick = jest.fn();
+  render(<AddBadgeListItem name={name} onClick={handleClick} />);
+  return { handleClick };
+};
+
+describe("AddBadgeListItem", () => {
+  it("renders badge with correct name", () => {
+    setup();
```

Comment:
are you sure about this? we have default props in so many of our other tests

Reply:
Maybe I'm on the extreme side, but I don't hate duplication in this case if it makes reading the test easier.

I'm cool if others thing otherwise though.